kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -53,19 +57,19 @@
 
     <table class="roster">
 
-      <caption class="visuallyhidden">Users</caption>
+      <caption class="visuallyhidden">{{$tr('users')}}</caption>
 
       <!-- Table Headers -->
       <thead v-if="usersMatchFilter">
         <tr>
-          <th class="col-header" scope="col"> Full Name </th>
+          <th class="col-header" scope="col"> {{$tr('fullName')}} </th>
           <th class="col-header" scope="col">
             <span class="role-header" aria-hidden="true">
-              Role
+              {{$tr('kind')}}
```

Comment:
If I recall, the roles are going to be limited to Admin or Learner soon (since Coach is only relevant in the contexts of a class). I don't know how much more accurate Role would be in that case. 

As for specificity, I think you might have a point there. "This user's role is..." sounds more appropriate than "This kind of user can...." in an organization. I was leaning `kind` because that's the terminology I used in actions and all that, and I used that word because `kind` is an attribute of `role` models. 

Anyway, I'm leaning `role` now. @radinamatic I think `type` falls along the same lines as `kind` semantically, so @rtibbles's argument would still hold.